cvmfs/quota_posix.cc

```diff
@@ -358,8 +362,8 @@ PosixQuotaManager *PosixQuotaManager::CreateShared(
   preserve_filedes.insert(2);
   preserve_filedes.insert(pipe_boot[1]);
   preserve_filedes.insert(pipe_handshake[0]);
-
-  retval = ManagedExec(command_line, preserve_filedes, map<int, int>(), false);
+  pid_t newcachemgr_pid;
+  retval = ManagedExec(command_line, preserve_filedes, map<int, int>(), false, false, true, &newcachemgr_pid);
```

Comment:
Probably it would be nicer to have a config struct as an input to that function, so that all the parameters are named. In the meantime, we can annotate with comments the names of at least the boolean parameters.